firebase/firebase.json

```diff
@@ -7,6 +7,26 @@
     "predeploy": [
       "npm --prefix \"$RESOURCE_DIR\" run lint",
       "npm --prefix \"$RESOURCE_DIR\" run build"
+    ],
+    "source": "functions"
+  },
+  "hosting": {
+    "public": "public",
```

Comment:
FYI- I set up "public" as a symlink to "../build" and now hosting serves it.  This isn't necessary for now, but it moves us towards being able to move the whole site to Firebase Hosting if we feel like it.  i.e. for a Firebase-hosted version of the site, see: https://covidactnow-dev.web.app/